src/NzbDrone.Core/Indexers/Definitions/Toloka.cs

```diff
@@ -408,6 +413,19 @@ public IList<ReleaseInfo> ParseResponse(IndexerResponse indexerResponse)
                     MinimumSeedTime = 0
                 };
 
+                if (row.QuerySelector("img[src=\"images/gold.gif\"]") != null)
```

Comment:
FYI: `images/authors.gif` also can be included here. Maybe I'll do id a bit later

Reply:
Seems like a translation issue. In Ukrainian `завантажене` means `downloaded`, but it can mean `uploaded` as well in some contexts.

I don't have any link to FAQ (using this tracker since 2014 but never read the rules, shame on me) but if you open any release with this status there is exact same text as for gold (diff in the 1st word only):
```
Золотий реліз (завантажене не впливає на рейтинг)
Авторський реліз (завантажене не впливає на рейтинг)
```